### DISCUSSION TITLE
question about vulnerabilities identified during Tatool installation

### DISCUSSION THREAD

**Discussion by dickinsonca** [tatool/tatool-web#167] [Q&A] [answered]
Hello.  I just installed Tattool web (after installing nodeJS and mongoDB), and during the installation, there was a message that a number of vulnerabilities (ranging from low to critical) were identified.  I'm assuming these are things that should be addressed, but this is the first time I have experienced something like this during an installation.  Any information about this would be greatly appreciated.  Thank you.

Chris Dickinson

**Comment by alocher** [comment, marked as answer]
Hi Chris,
Nothing to worry about. From time to time we are going through upgrading dependencies that have some identified vulnerabilities. Nothing you need to worry about in your local installation.
